app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarLayout.kt

```diff
@@ -698,7 +653,7 @@ open class OmnibarLayout @JvmOverloads constructor(
         tabsMenu.isVisible = newTransitionState.showTabsMenu
         fireIconMenu.isVisible = newTransitionState.showFireIcon
         browserMenu.isVisible = newTransitionState.showBrowserMenu
-        browserMenuHighlight.isVisible = newTransitionState.showBrowserMenuHighlight
+        browserMenuHighlight.isVisible = viewState.showBrowserMenuHighlight
```

Comment:
Is this change correct? The same value is used in the `newTransitionState` but I think we should keep the old code for consistency.